rocAL/rocAL_pybind/examples/rocAL_api_python_unittest.py

```diff
@@ -52,154 +77,322 @@ def main():
-            output1 = fn.rotate(images)
-            output2 = fn.fish_eye(output1)
-            output3 = fn.fog(output2)
-            pipe.set_outputs(output1, output2, output3)
-            output_set = 1
-        elif augmentation_name == "resize_brightness_jitter":
-            output1 = fn.resize(images, resize_x=300, resize_y=300)
-            output2 = fn.brightness(output1)
-            output3 = fn.jitter(output2)
-            pipe.set_outputs(output1, output2, output3)
-            output_set = 1
-        elif augmentation_name == "vignetter_blur":
-            output1 = fn.vignette(images)
-            output2 = fn.blur(output1)
-            pipe.set_outputs(output1, output2)
-            output_set = 1
-
-        if output_set==0:
-                pipe.set_outputs(output)
+            output = fn.copy(
+                images,
+                rocal_tensor_output_layout=tensor_layout,
+                rocal_tensor_output_datatype=tensor_dtype)
+        elif augmentation_name == "resize_crop_mirror":
+            output = fn.resize_crop_mirror(
+                images,
+                resize_height=400,
+                resize_width=400,
+                crop_h=200,
+                crop_w=200,
+                rocal_tensor_output_layout=tensor_layout,
+                rocal_tensor_output_datatype=tensor_dtype)
+        elif augmentation_name == "lens_correction":
+            output = fn.lens_correction(
+                images,
+                strength=2.9,
+                zoom=1.2,
+                rocal_tensor_output_layout=tensor_layout,
+                rocal_tensor_output_datatype=tensor_dtype)
+        elif augmentation_name == "blend":
+            output1 = fn.rotate(
+                images,
+                angle=45.0,
+                dest_width=640,
+                dest_height=480,
+                rocal_tensor_output_layout=tensor_layout,
+                rocal_tensor_output_datatype=tensor_dtype)
+            output = fn.blend(
+                images,
+                output1,
+                ratio=0.5,
+                rocal_tensor_output_layout=tensor_layout,
+                rocal_tensor_output_datatype=tensor_dtype)
+        elif augmentation_name == "resize_crop":
+            output = fn.resize_crop(
+                images,
+                resize_width=640,
+                resize_height=480,
+                crop_area_factor=0.25,
+                crop_aspect_ratio=1.2,
+                x_drift=0.6,
+                y_drift=0.4,
+                rocal_tensor_output_layout=tensor_layout,
+                rocal_tensor_output_datatype=tensor_dtype)
+        elif augmentation_name == "center_crop":
+            output = fn.center_crop(
+                images,
+                crop=[2, 224, 224],
+                rocal_tensor_output_layout=tensor_layout,
+                rocal_tensor_output_datatype=tensor_dtype)
+
+        if output_set == 0:
+            pipe.setOutputs(output)
```

Comment:
Please change to set_outputs

Reply:
Every function in pipeline.py follows camel case so changing this will make this look out of order